js/src/personal/kelly/Kelly.module.css

```diff
@@ -1,10 +1,15 @@
 .title {
-  color: light-dark(var(--mantine-color-black), var(--mantine-color-white));
-  font-size: rem(100px);
-  font-weight: 900;
-  letter-spacing: rem(-2px);
+  font-family: "Noto Serif Display", serif;
```

Comment:
Can you squash your commits?
Also could you add the linear id to your commit?

L=ENG-99

Reply:
Added